File: app/src/main/java/com/yieldzoneandroid/demo/ShowBannerActivity.java
package com.yieldzoneandroid.demo;

import android.content.Context;
import android.content.Intent;
import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.FrameLayout;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.yieldzone.ads.YieldzoneAdBannerAd;
import com.yieldzone.baselibrary.AdSize;
import com.yieldzone.baselibrary.YieldzoneLoadAdError;
import com.yieldzone.baselibrary.listeners.BannerAdListener;

import static com.yieldzoneandroid.demo.TestPidUtils.YZ_BANNER_ID;


public class ShowBannerActivity extends AppCompatActivity {
    private static final String TAG = com.yieldzoneandroid.demo.ShowBannerActivity.class.getSimpleName();

    private YieldzoneAdBannerAd bannerAd1;

    public static void start(Context context) {
        Intent intent = new Intent(context, com.yieldzoneandroid.demo.ShowBannerActivity.class);
        intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
        context.startActivity(intent);
    }

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_show_banner);
        //banner广告
        FrameLayout container = findViewById(R.id.adContainer);
        findViewById(R.id.btn1).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner load...");
                bannerAd1 =  new YieldzoneAdBannerAd(com.yieldzoneandroid.demo.ShowBannerActivity.this,new BannerAdListener() {
                    @Override
                    public void onBannerAdClosed() {
                        Log.d(TAG, "关闭广告");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner close...");
                    }

                    @Override
                    public void onBannerAdExpose() {
                        Log.d(TAG, "广告Expose");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner expose...");
                    }

                    @Override
                    public void onAdFailedToLoad(int errorCode) {
                        Log.d(TAG, "加载失败");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner failed...");
                    }

                    @Override
                    public void onAdFailedToLoad(YieldzoneLoadAdError error) {
                        Log.d(TAG, "加载失败");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner failed...");
                    }

                    @Override
                    public void onBannerAdOpened() {
                        Log.d(TAG, "展示广告opoen");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner opened...");
                    }

                    @Override
                    public void onBannerAdLoaded() {
                        Log.d(TAG, "广告加载成功");
                        bannerAd1.showBanner(container);
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner load success...");
                    }

                    @Override
                    public void onBannerAdClicked() {
                        Log.d(TAG, "广告被点击");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner click...");
                    }

                }, YZ_BANNER_ID, AdSize.BANNER_320_50);//3DBFEE43-501F-F6DE-3F56-01AE2F44BF40-1
                bannerAd1.loadAd();
            }
        });

        findViewById(R.id.btn2).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                bannerAd1 =  new YieldzoneAdBannerAd(com.yieldzoneandroid.demo.ShowBannerActivity.this,new BannerAdListener() {
                    @Override
                    public void onBannerAdClosed() {
                        Log.d(TAG, "关闭广告");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner close...");
                    }

                    @Override
                    public void onBannerAdExpose() {
                        Log.d(TAG, "广告Expose");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner expose...");
                    }

                    @Override
                    public void onAdFailedToLoad(int errorCode) {
                        Log.d(TAG, "加载失败");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner failed...");
                    }

                    @Override
                    public void onAdFailedToLoad(YieldzoneLoadAdError error) {
                        Log.d(TAG, "加载失败");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner failed...");
                    }

                    @Override
                    public void onBannerAdOpened() {
                        Log.d(TAG, "展示广告opoen");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner opened...");
                    }

                    @Override
                    public void onBannerAdLoaded() {
                        Log.d(TAG, "广告加载成功");
                        bannerAd1.showBanner(container);
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner load success...");
                    }

                    @Override
                    public void onBannerAdClicked() {
                        Log.d(TAG, "广告被点击");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner click...");
                    }

                }, YZ_BANNER_ID, AdSize.BANNER_320_100);//3DBFEE43-501F-F6DE-3F56-01AE2F44BF40-1
                bannerAd1.loadAd();
            }
        });

        findViewById(R.id.btn3).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                bannerAd1 =  new YieldzoneAdBannerAd(com.yieldzoneandroid.demo.ShowBannerActivity.this,new BannerAdListener() {
                    @Override
                    public void onBannerAdClosed() {
                        Log.d(TAG, "关闭广告");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner close...");
                    }

                    @Override
                    public void onBannerAdExpose() {
                        Log.d(TAG, "广告Expose");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner expose...");
                    }

                    @Override
                    public void onAdFailedToLoad(int errorCode) {
                        Log.d(TAG, "加载失败");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner failed...");
                    }

                    @Override
                    public void onAdFailedToLoad(YieldzoneLoadAdError error) {
                        Log.d(TAG, "加载失败");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner failed...");
                    }

                    @Override
                    public void onBannerAdOpened() {
                        Log.d(TAG, "展示广告opoen");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner opened...");
                    }

                    @Override
                    public void onBannerAdLoaded() {
                        Log.d(TAG, "广告加载成功");
                        bannerAd1.showBanner(container);
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner load success...");
                    }

                    @Override
                    public void onBannerAdClicked() {
                        Log.d(TAG, "广告被点击");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner click...");
                    }

                }, YZ_BANNER_ID, AdSize.BANNER_320_250);//3DBFEE43-501F-F6DE-3F56-01AE2F44BF40-1
                bannerAd1.loadAd();
            }
        });

        findViewById(R.id.btn4).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                bannerAd1 =  new YieldzoneAdBannerAd(com.yieldzoneandroid.demo.ShowBannerActivity.this,new BannerAdListener() {
                    @Override
                    public void onBannerAdClosed() {
                        Log.d(TAG, "关闭广告");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner close...");
                    }

                    @Override
                    public void onBannerAdExpose() {
                        Log.d(TAG, "广告Expose");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner expose...");
                    }

                    @Override
                    public void onAdFailedToLoad(int errorCode) {
                        Log.d(TAG, "加载失败");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner failed...");
                    }

                    @Override
                    public void onAdFailedToLoad(YieldzoneLoadAdError error) {
                        Log.d(TAG, "加载失败");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner failed...");
                    }

                    @Override
                    public void onBannerAdOpened() {
                        Log.d(TAG, "展示广告opoen");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner opened...");
                    }

                    @Override
                    public void onBannerAdLoaded() {
                        Log.d(TAG, "广告加载成功");
                        bannerAd1.showBanner(container);
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner load success...");
                    }

                    @Override
                    public void onBannerAdClicked() {
                        Log.d(TAG, "广告被点击");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner click...");
                    }

                }, YZ_BANNER_ID, AdSize.BANNER_468_60);//3DBFEE43-501F-F6DE-3F56-01AE2F44BF40-1
                bannerAd1.loadAd();
            }
        });

        findViewById(R.id.btn5).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                bannerAd1 =  new YieldzoneAdBannerAd(com.yieldzoneandroid.demo.ShowBannerActivity.this,new BannerAdListener() {
                    @Override
                    public void onBannerAdClosed() {
                        Log.d(TAG, "关闭广告");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner close...");
                    }

                    @Override
                    public void onBannerAdExpose() {
                        Log.d(TAG, "广告Expose");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner expose...");
                    }

                    @Override
                    public void onAdFailedToLoad(int errorCode) {
                        Log.d(TAG, "加载失败");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner failed...");
                    }

                    @Override
                    public void onAdFailedToLoad(YieldzoneLoadAdError error) {
                        Log.d(TAG, "加载失败");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner failed...");
                    }

                    @Override
                    public void onBannerAdOpened() {
                        Log.d(TAG, "展示广告opoen");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner opened...");
                    }

                    @Override
                    public void onBannerAdLoaded() {
                        Log.d(TAG, "广告加载成功");
                        bannerAd1.showBanner(container);
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner load success...");
                    }

                    @Override
                    public void onBannerAdClicked() {
                        Log.d(TAG, "广告被点击");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner click...");
                    }

                }, YZ_BANNER_ID, AdSize.BANNER_728_90);//3DBFEE43-501F-F6DE-3F56-01AE2F44BF40-1
                bannerAd1.loadAd();
            }
        });

        findViewById(R.id.btn6).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                bannerAd1 =  new YieldzoneAdBannerAd(com.yieldzoneandroid.demo.ShowBannerActivity.this,new BannerAdListener() {
                    @Override
                    public void onBannerAdClosed() {
                        Log.d(TAG, "关闭广告");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner close...");
                    }

                    @Override
                    public void onBannerAdExpose() {
                        Log.d(TAG, "广告Expose");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner expose...");
                    }

                    @Override
                    public void onAdFailedToLoad(int errorCode) {
                        Log.d(TAG, "加载失败");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner failed...");
                    }

                    @Override
                    public void onAdFailedToLoad(YieldzoneLoadAdError error) {
                        Log.d(TAG, "加载失败");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner failed...");
                    }

                    @Override
                    public void onBannerAdOpened() {
                        Log.d(TAG, "展示广告opoen");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner opened...");
                    }

                    @Override
                    public void onBannerAdLoaded() {
                        Log.d(TAG, "广告加载成功");
                        bannerAd1.showBanner(container);
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner load success...");
                    }

                    @Override
                    public void onBannerAdClicked() {
                        Log.d(TAG, "广告被点击");
                        YZToast.show(com.yieldzoneandroid.demo.ShowBannerActivity.this, "banner click...");
                    }

                }, YZ_BANNER_ID, AdSize.BANNER_320_90);//3DBFEE43-501F-F6DE-3F56-01AE2F44BF40-1
                bannerAd1.loadAd();
            }
        });
    }
}

File: appkt/src/main/java/com/yieldzoneandroid/myapplication/ShowBannerActivity.kt
package com.yieldzoneandroid.myapplication

import android.content.Context
import android.content.Intent
import android.os.Bundle
import android.util.Log
import androidx.appcompat.app.AppCompatActivity

import com.yieldzone.ads.YieldzoneAdBannerAd
import com.yieldzone.baselibrary.AdSize
import com.yieldzone.baselibrary.YieldzoneLoadAdError
import com.yieldzone.baselibrary.listeners.BannerAdListener
import com.yieldzoneandroid.myapplication.R
import kotlinx.android.synthetic.main.activity_show_banner.*

class ShowBannerActivity : AppCompatActivity() {

    var banner1 :YieldzoneAdBannerAd? = null

    companion object {
        fun start(context: Context) {
            val intent: Intent = Intent(context, ShowBannerActivity::class.java)
            intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
            context.startActivity(intent)
        }
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_show_banner)
        btn1.setOnClickListener {
            banner1 = YieldzoneAdBannerAd(
                this@ShowBannerActivity,
                object : BannerAdListener {
                    override fun onBannerAdClosed() {
                        Log.d("ShowBannerActivity", "关闭广告")
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner close..."
                        )
                    }

                    override fun onBannerAdExpose() {
                        Log.d(
                            "ShowBannerActivity",
                            "广告Expose"
                        )
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner expose..."
                        )
                    }

                    override fun onAdFailedToLoad(errorCode: Int) {
                        Log.d("ShowBannerActivity", "加载失败")
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner failed..."
                        )
                    }

                    override fun onAdFailedToLoad(error: YieldzoneLoadAdError) {
                        Log.d("ShowBannerActivity", "加载失败")
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner failed..."
                        )
                    }

                    override fun onBannerAdOpened() {
                        Log.d(
                            "ShowBannerActivity",
                            "展示广告opoen"
                        )
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner opened..."
                        )
                    }

                    override fun onBannerAdLoaded() {
                        Log.d(
                            "ShowBannerActivity",
                            "广告加载成功"
                        )
                        banner1?.showBanner(adContainer)
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner load success..."
                        )
                    }

                    override fun onBannerAdClicked() {
                        Log.d(
                            "ShowBannerActivity",
                            "广告被点击"
                        )
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner click..."
                        )
                    }
                },
                "3DBFEE43-501F-F6DE-3F56-01AE2F44BF40-1",
                AdSize.BANNER_320_50
            ) //3DBFEE43-501F-F6DE-3F56-01AE2F44BF40-1
            banner1?.loadAd();
        }
        btn2.setOnClickListener {
            banner1 = YieldzoneAdBannerAd(
                this@ShowBannerActivity,
                object : BannerAdListener {
                    override fun onBannerAdClosed() {
                        Log.d("ShowBannerActivity", "关闭广告")
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner close..."
                        )
                    }

                    override fun onBannerAdExpose() {
                        Log.d(
                            "ShowBannerActivity",
                            "广告Expose"
                        )
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner expose..."
                        )
                    }

                    override fun onAdFailedToLoad(errorCode: Int) {
                        Log.d("ShowBannerActivity", "加载失败")
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner failed..."
                        )
                    }

                    override fun onAdFailedToLoad(error: YieldzoneLoadAdError) {
                        Log.d("ShowBannerActivity", "加载失败")
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner failed..."
                        )
                    }

                    override fun onBannerAdOpened() {
                        Log.d(
                            "ShowBannerActivity",
                            "展示广告opoen"
                        )
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner opened..."
                        )
                    }

                    override fun onBannerAdLoaded() {
                        Log.d(
                            "ShowBannerActivity",
                            "广告加载成功"
                        )
                        banner1?.showBanner(adContainer)
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner load success..."
                        )
                    }

                    override fun onBannerAdClicked() {
                        Log.d(
                            "ShowBannerActivity",
                            "广告被点击"
                        )
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner click..."
                        )
                    }
                },
                "3DBFEE43-501F-F6DE-3F56-01AE2F44BF40-1",
                AdSize.BANNER_320_100
            ) //3DBFEE43-501F-F6DE-3F56-01AE2F44BF40-1
        }
        btn3.setOnClickListener {
            banner1 = YieldzoneAdBannerAd(
                this@ShowBannerActivity,
                object : BannerAdListener {
                    override fun onBannerAdClosed() {
                        Log.d("ShowBannerActivity", "关闭广告")
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner close..."
                        )
                    }

                    override fun onBannerAdExpose() {
                        Log.d(
                            "ShowBannerActivity",
                            "广告Expose"
                        )
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner expose..."
                        )
                    }

                    override fun onAdFailedToLoad(errorCode: Int) {
                        Log.d("ShowBannerActivity", "加载失败")
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner failed..."
                        )
                    }

                    override fun onAdFailedToLoad(error: YieldzoneLoadAdError) {
                        Log.d("ShowBannerActivity", "加载失败")
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner failed..."
                        )
                    }

                    override fun onBannerAdOpened() {
                        Log.d(
                            "ShowBannerActivity",
                            "展示广告opoen"
                        )
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner opened..."
                        )
                    }

                    override fun onBannerAdLoaded() {
                        Log.d(
                            "ShowBannerActivity",
                            "广告加载成功"
                        )
                        banner1?.showBanner(adContainer)
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner load success..."
                        )
                    }

                    override fun onBannerAdClicked() {
                        Log.d(
                            "ShowBannerActivity",
                            "广告被点击"
                        )
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner click..."
                        )
                    }
                },
                "3DBFEE43-501F-F6DE-3F56-01AE2F44BF40-1",
                AdSize.BANNER_320_250
            ) //3DBFEE43-501F-F6DE-3F56-01AE2F44BF40-1
        }
        btn4.setOnClickListener {
            banner1 = YieldzoneAdBannerAd(
                this@ShowBannerActivity,
                object : BannerAdListener {
                    override fun onBannerAdClosed() {
                        Log.d("ShowBannerActivity", "关闭广告")
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner close..."
                        )
                    }

                    override fun onBannerAdExpose() {
                        Log.d(
                            "ShowBannerActivity",
                            "广告Expose"
                        )
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner expose..."
                        )
                    }

                    override fun onAdFailedToLoad(errorCode: Int) {
                        Log.d("ShowBannerActivity", "加载失败")
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner failed..."
                        )
                    }

                    override fun onAdFailedToLoad(error: YieldzoneLoadAdError) {
                        Log.d("ShowBannerActivity", "加载失败")
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner failed..."
                        )
                    }

                    override fun onBannerAdOpened() {
                        Log.d(
                            "ShowBannerActivity",
                            "展示广告opoen"
                        )
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner opened..."
                        )
                    }

                    override fun onBannerAdLoaded() {
                        Log.d(
                            "ShowBannerActivity",
                            "广告加载成功"
                        )
                        banner1?.showBanner(adContainer)
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner load success..."
                        )
                    }

                    override fun onBannerAdClicked() {
                        Log.d(
                            "ShowBannerActivity",
                            "广告被点击"
                        )
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner click..."
                        )
                    }
                },
                "3DBFEE43-501F-F6DE-3F56-01AE2F44BF40-1",
                AdSize.BANNER_468_60
            ) //3DBFEE43-501F-F6DE-3F56-01AE2F44BF40-1
        }
        btn5.setOnClickListener {
            banner1 = YieldzoneAdBannerAd(
                this@ShowBannerActivity,
                object : BannerAdListener {
                    override fun onBannerAdClosed() {
                        Log.d("ShowBannerActivity", "关闭广告")
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner close..."
                        )
                    }

                    override fun onBannerAdExpose() {
                        Log.d(
                            "ShowBannerActivity",
                            "广告Expose"
                        )
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner expose..."
                        )
                    }

                    override fun onAdFailedToLoad(errorCode: Int) {
                        Log.d("ShowBannerActivity", "加载失败")
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner failed..."
                        )
                    }

                    override fun onAdFailedToLoad(error: YieldzoneLoadAdError) {
                        Log.d("ShowBannerActivity", "加载失败")
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner failed..."
                        )
                    }

                    override fun onBannerAdOpened() {
                        Log.d(
                            "ShowBannerActivity",
                            "展示广告opoen"
                        )
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner opened..."
                        )
                    }

                    override fun onBannerAdLoaded() {
                        Log.d(
                            "ShowBannerActivity",
                            "广告加载成功"
                        )

                        banner1?.showBanner(adContainer)
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner load success..."
                        )
                    }

                    override fun onBannerAdClicked() {
                        Log.d(
                            "ShowBannerActivity",
                            "广告被点击"
                        )
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner click..."
                        )
                    }
                },
                "3DBFEE43-501F-F6DE-3F56-01AE2F44BF40-1",
                AdSize.BANNER_728_90
            ) //3DBFEE43-501F-F6DE-3F56-01AE2F44BF40-1
        }
        btn6.setOnClickListener {
            banner1 = YieldzoneAdBannerAd(
                this@ShowBannerActivity,
                object : BannerAdListener {
                    override fun onBannerAdClosed() {
                        Log.d("ShowBannerActivity", "关闭广告")
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner close..."
                        )
                    }

                    override fun onBannerAdExpose() {
                        Log.d(
                            "ShowBannerActivity",
                            "广告Expose"
                        )
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner expose..."
                        )
                    }

                    override fun onAdFailedToLoad(errorCode: Int) {
                        Log.d("ShowBannerActivity", "加载失败")
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner failed..."
                        )
                    }

                    override fun onAdFailedToLoad(error: YieldzoneLoadAdError) {
                        Log.d("ShowBannerActivity", "加载失败")
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner failed..."
                        )
                    }

                    override fun onBannerAdOpened() {
                        Log.d(
                            "ShowBannerActivity",
                            "展示广告opoen"
                        )
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner opened..."
                        )
                    }

                    override fun onBannerAdLoaded() {
                        Log.d(
                            "ShowBannerActivity",
                            "广告加载成功"
                        )

                        banner1?.showBanner(adContainer)
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner load success..."
                        )
                    }

                    override fun onBannerAdClicked() {
                        Log.d(
                            "ShowBannerActivity",
                            "广告被点击"
                        )
                        YZToast.show(
                            this@ShowBannerActivity,
                            "banner click..."
                        )
                    }
                },
                "3DBFEE43-501F-F6DE-3F56-01AE2F44BF40-1",
                AdSize.BANNER_320_90
            ) //3DBFEE43-501F-F6DE-3F56-01AE2F44BF40-1
        }
    }
}